### ISSUE TITLE
Hello, is this still a thing?

### ISSUE THREAD

**Issue by Hexality** [stupefaction02/FlashVoltex#1]
Hey, I loved the concept of it and wanted to know if this will be developed in the future, since it seems promising and it would be awesome to have in osu.
~`(I actually don't have that much hope knowing the fact that this repo has been dead for like 10 months already.)`~

**Comment by stupefaction02** [comment]
Yeah. I want to get back to work at this summer's end. I was too busy with
my current job 😣. Also, if you have any ideas or concepts, or something
that you want to see in this project, feel free to share with me.

**Comment by Hexality** [comment]
I wanted more is to ask if it will support ksh (K-Shot Mania Chart file) format, cause idk how extensible is osu!framework inside the game itself when it comes to handle custom file extensions.

Also, osu!lazer supports native portrait mode so I think it would be cool if it was possible, not now but in the future, to use skins like the one used in Unnamed SDVX Clone like this one that simulates sdvx (talking about the gameplay itself) https://youtu.be/UFb3VSbDD6M?t=38

**Comment by NekomusuMae** [comment]
> I wanted more is to ask if it will support ksh (K-Shot Mania Chart file) format, cause idk how extensible is osu!framework inside the game itself when it comes to handle custom file extensions

It will not likely support it built in. Yoso (ADoFaI for osu!) is getting around this with a converter program that convert ADoFaI level files to osu! map files.
